.github/workflows/github-releases-to-discord.yml

```
@@ -0,0 +1,22 @@
+# https://github.com/SethCohen/github-releases-to-discord
+# on:
+#   release:
+#     types: [published]
+on: workflow_dispatch
+
+jobs:
+  github-releases-to-discord:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: GitHub Releases to Discord
+        uses: SethCohen/github-releases-to-discord@v1
```

```suggestion
        uses: SethCohen/github-releases-to-discord@b96a33520f8ad5e6dcdecee6f1212bdf88b16550
```

I'd use exact ref versioning on unpopular github actions, to prevent hijacking issues...